scripts/lib/post.js

```diff
@@ -6,27 +6,38 @@
       enctype: 'text/plain',
       fields: []
     }
+    body = body.trim();
+    if (!body.includes('=')){
+      const trashName = Math.random().toString(36).slice(2)
+      let trash;
+      if (body.at(-1) === "}"){
+        trash = `,"${trashName}":"="}`;
+      } else if (body.at(-1) === "]") {
+        trash = `,"${trashName}="]`;
+      } else {
+        throw new Error("Your body doesn't contain `=`, we can't help you POST it as a JSON now :(");
+      }
 
-    const trashName = Math.random().toString(36).slice(2)
-    const json = JSON.parse(body)
-
-    if (Array.isArray(json)) {
-      json.push(JSON.parse(`{ "${trashName}": "=" }`))
+      const stringified = body.slice(0,-1) + trash
```

Comment:
Sorry for requesting changes again.

If it is possible, I wish to use single-quote first, then double-quote.

Therefore, `"}"` should be `'}'`.

In addition, there should be a space after comma.

I would like to trouble you to check them again, thank you.

Reply:
@lebr0nli 

Hello, could you take above-mentioned recommendation into account and make changes again? Thank you.

Reply:
Hi, sorry for the late reply, I miss the message :(
I'm a little bit busy recently, but I will fix them in my free time, maybe this or next weekend.
By the way, I will also add some XSS payloads for CSP bypass with `AngularJS` at that time!

Reply:
It's now fixed!

Reply:
Looks great!

I will merge this PR and release new version ASAP.

Thanks for your contribution again.